File: uHunt/C++src/easy/LanguageDetection.cpp
using namespace std;
#include<cstdio>
#include<iostream>
#include<string>

bool test=true;

int main()
{
    if(test) freopen("../data/easy/LanguageDetection.in","r",stdin);

    string s;cin>>s;
    cout<<s<<endl;
    for(int i=1; i<10 ;i++)
    {
        printf("Case %d: ",i);
        if(s=="HELLO") printf("ENGLISH\n");
        else if(s=="HOLA") printf("SPANISH\n");
		else if(s=="HALLO") printf("GERMAN\n");
		else if(s=="BONJOUR") printf("FRENCH\n");
        else if(s=="CIAO") printf("ITALIAN\n");
        else if(s=="ZDRAVSTVUJTE") printf("RUSSIAN\n");
		else printf("UNKNOWN\n");
		cout << s<<endl;

		cin>>s;
    }
    return 0;
}

File: uHunt/src/easy/LanguageDetection.java
package easy;

import java.io.BufferedReader;
import java.io.File;
import java.io.FileReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class LanguageDetection
{
	private static BufferedReader br;
	private static boolean test=true;

	public static void main(String[] args) throws IOException 
	{
		if(test)br=new BufferedReader(new FileReader(new File("./data/easy/LanguageDetection.in")));
		else br=new BufferedReader(new InputStreamReader(System.in));
		solve();
		br.close();
	}

	private static void solve() throws IOException
	{
		String linea=br.readLine(), rta="";
		
		for(int i=1;!linea.equals("#");i++)
		{
			if(linea.contains("HELLO"))rta="ENGLISH";
			else if(linea.contains("HOLA"))rta="SPANISH";
			else if(linea.contains("HALLO"))rta="GERMAN";
			else if(linea.contains("BONJOUR"))rta="FRENCH";
			else if(linea.contains("CIAO"))rta="ITALIAN";
			else if(linea.contains("ZDRAVSTVUJTE"))rta="RUSSIAN";
			else rta="UNKNOWN";
			
			System.out.println("Case "+i+": "+rta);
			linea=br.readLine();
		}
	}
	
}